### PR TITLE
adding 'organize data' module

### PR DESCRIPTION
**Pull request by kthrog**

### CODE REVIEW

File: organize_data/organize_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    "Now we're ready to create a new folder in our sandbox project, and add a new file to it. Add the synID of your Synapse sandbox project between the quotes below. Not sure what a synID is, or how to find it? Use this [doc](https://docs.synapse.org/articles/getting_started.html#synapse-ids) to learn more."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mySandbox = syn.get('')"
```
Comment (mfazza) [inline organize_data/organize_data.ipynb]: I have a location for us to create folders now: `syn25488570` (:

File: organize_data/organize_data.ipynb [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`syn.get` retrieves an entity from Synapse, in this case, an entire project. ([Learn more.](https://python-docs.synapse.org/build/html/index.html?highlight=syn%20get#synapseclient.Synapse.get)) Note that you can also retrieve other entities, such as folders and individual files.\n",
```
Comment (mfazza) [inline organize_data/organize_data.ipynb]: If we go with the Syn id above then the wording here needs to change 🤷🏼‍♀️

**Review by mfazza — CHANGES_REQUESTED** [review]
Wow, this is really good @kthrog .  Since I created the Synapse Interactive Onboarding folder, I proposed a couple of changes to this notebook!  Thanks for piecing this together 🥇

**Comment by kthrog** [comment]
Great proposed changes -- I'll get to these in a bit, hopefully later this week!

**Comment by kthrog** [comment]
Okay, @mattfazza -- this should be all set! I made your changes, but looks like I can't actually merge the PR as I don't have write access to the repo. However, please feel free to merge this in.

**Pull request merged**